### ISSUE TITLE
macOS: keepassxc-cli libquazip error

### ISSUE THREAD

**Issue by eddies** [keepassxreboot/keepassxc#3270]
[TIP]:  # ( Provide a general summary of the issue in the title above ^^ )
[TIP]:  # ( DO NOT include screenshots of your actual database! )

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
Launch `keepassxc-cli` without an error, e.g.:

<details>
  <summary><code>$ keepassxc-cli -h</code></summary>

```console
Usage: keepassxc-cli [options] command
KeePassXC command line interface.

Available commands:
add            Add a new entry to a database.
clip           Copy an entry's password to the clipboard.
create         Create a new database.
diceware       Generate a new random diceware passphrase.
edit           Edit an entry.
estimate       Estimate the entropy of a password.
extract        Extract and print the content of a database.
generate       Generate a new random password.
locate         Find entries quickly.
ls             List database entries.
merge          Merge two databases.
rm             Remove an entry from the database.
show           Show an entry's information.


Options:
  --debug-info   Displays debugging information.
  -h, --help     Displays this help.
  -v, --version  Displays version information.

Arguments:
  command        Name of the command to execute.
```

</details>

## Current Behavior
[NOTE]: # ( Tell us what actually happens )
```console
$ keepassxc-cli -h
dyld: Library not loaded: /usr/local/opt/quazip/lib/libquazip.1.dylib
  Referenced from: /usr/local/bin/keepassxc-cli
  Reason: image not found
Abort trap: 6
```

## Possible Solution
[NOTE]: # ( Not required, but suggest a fix/reason for the bug )
Possibly related to #3088?

## Steps to Reproduce
[NOTE]: # ( Provide a link to a live example, or an unambiguous set of steps to )
[NOTE]: # ( reproduce this bug. Include code to reproduce, if relevant )
1. Install 2.4.3 for macOS
2. run `keepassxc-cli`

## Context
[NOTE]: # ( How has this issue affected you? What unique circumstances do you have? )
I have verified that I can successfully run keepassxc-cli with versions 2.4.0 and 2.4.1. I have only seen this error appear with 2.4.3.

## Debug Info
```
KeePassXC - Version 2.4.3
Revision: 5d6ef0c

Qt 5.12.3
Debugging mode is disabled.

Operating system: macOS Mojave (10.14)
CPU architecture: x86_64
Kernel: darwin 18.6.0

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey
- TouchID

Cryptographic libraries:
 libgcrypt 1.8.4
```

**Comment by droidmonkey** [comment]
Sigh, this is the same issue that hit the proxy in 2.4.2. In the meantime you can install homebrew and then use it to install libquazip.

**Comment by seanbreckenridge** [comment]
Had the same error as above.

Doing `brew install quazip` allows me to use `keepassxc-cli` successfully, though there is what seems to be a warning when running:

```

❯ keepassxc-cli 
objc[53857]: Class QMacAutoReleasePoolTracker is implemented in both /usr/local/opt/qt/lib/QtCore.framework/Versions/5/QtCore (0x10e3a0090) and /Applications/KeePassXC.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore (0x10ddaf090). One of the two will be used. Which one is undefined.
objc[53857]: Class QT_ROOT_LEVEL_POOL__THESE_OBJECTS_WILL_BE_RELEASED_WHEN_QAPP_GOES_OUT_OF_SCOPE is implemented in both /usr/local/opt/qt/lib/QtCore.framework/Versions/5/QtCore (0x10e3a0108) and /Applications/KeePassXC.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore (0x10ddaf108). One of the two will be used. Which one is undefined.
objc[53857]: Class RunLoopModeTracker is implemented in both /usr/local/opt/qt/lib/QtCore.framework/Versions/5/QtCore (0x10e3a0130) and /Applications/KeePassXC.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore (0x10ddaf130). One of the two will be used. Which one is undefined.
Usage: keepassxc-cli [options] command
KeePassXC command line interface.

...
```

**Comment by phoerious** [comment]
Perhaps we should add a test for this to the CI that checks all binaries for linked libraries from the brew directory.

**Comment by stephen-dexda** [comment]
Is there a chance of a 2.4.4 or 2.4.3.1 to fix this?

**Comment by droidmonkey** [comment]
No this will be fixed when 2.5.0 is released. There is an easy workaround mentioned above.
